k8s_itests/scripts/setup.sh

```diff
@@ -7,3 +7,9 @@ mkdir $PAASTA_CONFIG_DIR
 
 echo "Applying kubernetes resources"
 kubectl apply -Rf $KUBE_RESOURCE_DIR --context $KIND_CLUSTER
+
+
+
+ZOOKEEPER_PORT=$(shuf -i 2000-65000 -n 1)
```

Comment:
hmm, this is tricky: I think we have some helpers internally for picking an ephemeral port so that we don't collide with any of the other zillion things that need a port on our boxes - not sure if that's been open-sourced or is small enough to vendor some of it in this repo

Reply:
oh, we do have this available in this repo: [pick_random_port](https://github.com/Yelp/paasta/blob/2e1cafdc687f231eca07388f468bf4b00ebb95cd/paasta_tools/cli/utils.py#L1032) is what we likely want to do here

Reply:
this was committed by mistake 😭😭😭😭

Reply:
This is also what broke the test 😭😭😭😭😭